nosql/1.0/_index.md

```diff
@@ -46,12 +46,13 @@ template.delete(Car.class, 1L);
 
 
 * [Jakarta NoSQL Release Plan](https://projects.eclipse.org/projects/ee4j.nosql/governance)
-* [Jakarta NoSQL 1.0 Specification Document](./nosql-1.0.0-M1.pdf) (PDF)
-* [Jakarta NoSQL 1.0 Specification Document](./nosql-1.0.0-M1.html) (HTML)
+* [Jakarta NoSQL 1.0 Specification Document](./jakarta-nosql-1.0.0.pdf) (PDF)
+* [Jakarta NoSQL 1.0 Specification Document](./jakarta-nosql-1.0.0.html) (HTML)
 * [Jakarta NoSQL 1.0 Specification Javadoc](./apidocs)
 * Maven coordinates
-  * [jakarta-nosql-api:jakarta.nosql:jar:1.0.0-M1](https://repo1.maven.org/maven2/jakarta/nosql/jakarta.nosql-api/1.0.0-M1/)
-
+  * [jakarta-nosql:jakarta.nosql-api:jar:1.0.0](https://repo1.maven.org/maven2/jakarta/nosql/jakarta.nosql-api/1.0.0)
```

Comment:
While this is the target address to be used once this process is complete, I believe the binaries of 1.0.0 should be staged https://jakarta.oss.sonatype.org/content/groups/staging/jakarta/nosql/jakarta.nosql-api/ for reviewers to review.

Reply:
Good point, @kazumura. Could you confirm this?

Reply:
@otaviojava 
You need to write the URL of the final maven central location at "_inex.md". So "https://repo1.maven.org/maven2/jakarta/nosql/jakarta.nosql-api/1.0.0" is good one.
On the other hand, the staged URL must be specified in the description of this PR.
For now, "https://jakarta.oss.sonatype.org/content/groups/staging/jakarta/nosql/jakarta.nosql-api/" is specified in the desciption, but I cannot access to this URL. Please use the location where anyone can access for reviewing staging binaries.

Reply:
@kazumura, the documentation goes until the stagging build. Do you know how to do it?